crates/rooch-framework-tests/tests/cases/private_generics/verify_private_generics_in_primitive_types.exp

```diff
@@ -0,0 +1,121 @@
+   ┌─ /tmp/tempfile:17:9
+   │
+17 │         bar<u16>();
+   │         ^^^^^^^^^^
+
+error: resource type "U256" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:29:9
+   │
+29 │         bar<u256>();
+   │         ^^^^^^^^^^^
+
+error: resource type "U32" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:20:9
+   │
+20 │         bar<u32>();
+   │         ^^^^^^^^^^
+
+error: resource type "U64" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:23:9
+   │
+23 │         bar<u64>();
+   │         ^^^^^^^^^^
+
+error: resource type "U8" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:14:9
+   │
+14 │         bar<u8>();
+   │         ^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:65:9
+   │
+65 │         bar<vector<StructT>>();
+   │         ^^^^^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:59:9
+   │
+59 │         bar<vector<address>>();
+   │         ^^^^^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:38:9
+   │
+38 │         bar<vector<bool>>();
+   │         ^^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:62:9
+   │
+62 │         bar<vector<signer>>();
+   │         ^^^^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:53:9
+   │
+53 │         bar<vector<u128>>();
+   │         ^^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:44:9
+   │
+44 │         bar<vector<u16>>();
+   │         ^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:56:9
+   │
+56 │         bar<vector<u256>>();
+   │         ^^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:47:9
+   │
+47 │         bar<vector<u32>>();
+   │         ^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:50:9
+   │
+50 │         bar<vector<u64>>();
+   │         ^^^^^^^^^^^^^^^^^^
+
+error: resource type "Vector" in function "0x42::test::bar" not defined in current module or not allowed
+   ┌─ /tmp/tempfile:41:9
+   │
+41 │         bar<vector<u8>>();
+   │         ^^^^^^^^^^^^^^^^^
+
+status EXECUTED
```

Comment:
If the verifier raises an error, the module should not publish success, and the status is not `EXECUTED` @steelgeek091

Reply:
Fxied in PR https://github.com/rooch-network/rooch/pull/345

Reply:
Fixed this PR.